pkg/model/fakes.go

```diff
@@ -65,13 +66,21 @@ func (fe *FakeEnvironment) AppendCPUUtilization(cpu *simulator.CPUUtilization) {
 	fe.TheCPUUtilizations = append(fe.TheCPUUtilizations, cpu)
 }
 
+func NewFakeEnvironment() *FakeEnvironment {
+	return &FakeEnvironment{
+		ThePlugin: NewFakePluginPartition(),
+	}
+}
+
 type FakeReplica struct {
```

Comment:
Probably not for this PR, but why do we need FakeReplica, what stops us from using real Replica? 

In general, it's better to use real objects in tests whenever possible, instead of mocks and fakes. This way the tests execute code in a way that's as close as possible to the way the code is executed in reality.

Reply:
It's a good point! FakeReplica is needed to stub activate() and deactivate() methods in replica because there we call kubernetes  stuff, but for unit testing we need just check that this methods were called in replica. But once I remove all knative stuff from skenario in another PR, most probably we don't need to mock replica anymore we can use a real one.
But anyway, it's part of another PR.